classes/Store.php

```diff
@@ -136,13 +139,14 @@ public function __construct($idStore = null, $idLang = null)
      *
      * @return array|false|mysqli_result|null|PDOStatement|resource
      */
-    public static function getStores()
+    public static function getStores($idLang)
```

Comment:
Changing this method signature is a compatibility break that should be noted on the PR description.

Also, please write a proper PHPDoc for this new parameter.